### DISCUSSION TITLE
2D top down car physics demo

### DISCUSSION THREAD

**Discussion by aventius-software** [MonoGame/MonoGame#8512] [Showcase]
Just a small little demo using Monogame and Aether physics with a arcade style car controller. Nothing special or fancy, but anyway 🫡

**Comment by aventius-software** [comment]
Oops, forgot to put a link... https://github.com/aventius-software/Monogame/tree/master/TopDownCarPhysics

![FacepalmPicardGIF](https://github.com/user-attachments/assets/1c35183e-d3de-44b4-884f-3e8fcd65d9ee)

**Comment by mandala-dev** [comment]
That's a very cool demo, making a Super Sprint-like game would be much easier using your code.

**Reply by aventius-software** [reply]
Thanks, glad you liked it... I'm planning to add more little 'demo' projects like this to the repo eventually
